src/rules/no-unused-modules.ts

```diff
@@ -306,7 +299,7 @@ let srcFiles: Set<string>
 let lastPrepareKey: string
 
 const doPreparation = (
-  src: string[] = [],
```

Comment:
This is the actual regression fix. By initializing `src` to an empty array, it was no longer falsy, which would cause `getSrc()` to incorrectly return it as-is instead of defaulting it to `process.cwd()`.